hummingbot/client/hummingbot_application.py

```diff
@@ -1141,6 +1142,99 @@ def compare_balance_snapshots(self):
         lines = ["", "  Performance:"] + ["    " + line for line in str(df).split("\n")]
         self._notify("\n".join(lines))
 
+    def add_balances(self, market_symbol_pair, is_base, is_starting, base, quote):
+        """ Adds the balance of either the base or the quote in the given market symbol pair token to the corresponding
+        tuple pair.
+
+        :param market_symbol_pair: type market_symbol_pair
```

Comment:
The modern way to do type annotation in Python >= 3.5 is with the ':' notation inside the function arguments. You can still keep the text description here, but type annotation should be in the arguments list now.

Reply:
Also.. the type annotation for market_symbol_pair is invalid.